jenkins/sudoersd.sls

```diff
@@ -0,0 +1,13 @@
+sudo-pkg-for-jenkins:
+  pkg:
+    - name: sudo
+    - installed
+
+jenkins-sudoersd:
```

Comment:
I wonder whether we should just create an actual file in the state directory and drive the content from there as oppose to driving it via ```contents```. Open to discussion.